tests/filecheck/dialects/memref/memref_ops.mlir

```diff
@@ -17,8 +17,10 @@ builtin.module {
     %3 = arith.constant 42 : index
     "memref.store"(%3, %2, %1) : (index, memref<1xindex>, index) -> ()
     "memref.store"(%3, %2, %1) <{"nontemporal" = true}> : (index, memref<1xindex>, index) -> ()
+    "memref.store"(%3, %2, %1) <{"nontemporal" = false}> : (index, memref<1xindex>, index) -> ()
     %4 = "memref.load"(%2, %1) : (memref<1xindex>, index) -> index
     %f = "memref.load"(%2, %1) <{"nontemporal" = false}> : (memref<1xindex>, index) -> index
+    %g = "memref.load"(%2, %1) <{"nontemporal" = true}> : (memref<1xindex>, index) -> index
```

Comment:
while you're in this file, could you please migrate the inputs to use custom syntax instead of generic? will make them easier to read